Dockerfile.build

```diff
@@ -28,13 +34,15 @@ ARG JOBS
 # + libelf-dev to build the kernel modules
 # Tools and libraries for p4lang/target-syslibs and p4lang/target-utils
 # + cmake libedit-dev expat
-ENV PKG_DEPS pkg-config zip zlib1g-dev unzip python wget ca-certificates \
+# Tools to patch the SONiC kernel headers
+# + rsync
```

Comment:
@pudelkoM I think we also need to install `linux-kbuild-4.19` to get `scripts` and `tools` dir

Reply:
I think one of the header debs depends on it and pulls it in automatically:

```
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  linux-compiler-gcc-8-x86 linux-kbuild-4.19
The following NEW packages will be installed:
  linux-compiler-gcc-8-x86 linux-headers-4.19.0-12-2-amd64 linux-kbuild-4.19
0 upgraded, 3 newly installed, 0 to remove and 6 not upgraded.
```

Reply:
we can still install it explicitly

Reply:
oh, NVM. 
I was using dpkg to install it which won't install dependencies by default